src/overlays/actors/ovl_Door_Ana/z_door_ana.c

```diff
@@ -21,20 +24,169 @@ const ActorInit Door_Ana_InitVars = {
     (ActorFunc)DoorAna_Update,
     (ActorFunc)DoorAna_Draw
 };
-*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Door_Ana_0x808E01A0/func_808E01A0.asm")
+static ColliderCylinderInit sCylinderInit = { 
+    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_NONE, OC2_NONE, COLSHAPE_CYLINDER, },
+    { ELEMTYPE_UNK2, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
+    { 50, 10, 0, { 0, 0, 0 } },
+};
+
+// entrances grottos can link to statically, type DOORANA_TYPE_ADJACENT uses scene exit addresses instead
+static u16 entrances[] = {
```

Comment:
I feel that it would be helpful to have a `ENTRANCE` macro in `macros.h`, to pack these values together. Maybe something based on `Scene_CreateEntrance`, `#define ENTRANCE(sceneIndex, spawnIndex, offset) (((sceneIndex << 9) | (spawnIndex << 4)) | offset)`

Is there something already in OOT?

Reply:
Turns out OOT entrances worked a different way, so this would be a new macro for MM